### PR TITLE
ACMS-1659: Drush command to create headless user.

### PR DESCRIPTION
**Pull request by apathak18**
**Motivation**

Fixes ACMS-1659

**Proposed changes**
Global drush command to create headless user.

### CODE REVIEW

**Review by chandan-singh7929 — APPROVED** [review]

File: modules/acquia_cms_common/acquia_cms_common.install [hunk 1]
```diff
@@ -84,6 +85,10 @@ function acquia_cms_common_install($is_syncing) {
       'purge_ui',
     ]);
   }
+  else {
+    // Set acquia_cms_headless status variable.
```
Comment (chandan-singh7929) [inline modules/acquia_cms_common/acquia_cms_common.install:89]: Change this message, instead of acquia_cms_headless use acquia_cms sync status may be.

Comment (apathak18) [reply modules/acquia_cms_common/acquia_cms_common.install]: Done

**Review by chandan-singh7929 — APPROVED** [review]
Overall changes looks good to me, there are some minor comments need to be updated.

File: modules/acquia_cms_common/assets/scaffold/files/PostConfigurationCommands.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drush\Commands;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Consolidation\AnnotatedCommand\CommandResult;
+use Drupal\acquia_cms_common\Event\PostConfigEvent;
+
+/**
+ * This handles post configurations operation.
```
[the remark this answers is not in the document]
Comment (rajeshreeputra) [reply modules/acquia_cms_common/assets/scaffold/files/PostConfigurationCommands.php:10]: +1

File: modules/acquia_cms_common/assets/scaffold/files/PostConfigurationCommands.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  /**
+   * Process the dispatch of event as per configuration operation.
+   *
+   * @param string $type
+   *   Value of configuration.
+   */
+  protected function processDispatchEvents(string $type): void {
+    $eventObject = new PostConfigEvent();
+    $eventName = PostConfigEvent::ACMS_POST_CONFIG_IMPORT;
```
[the remark this answers is not in the document]
Comment (chandan-singh7929) [reply modules/acquia_cms_common/assets/scaffold/files/PostConfigurationCommands.php:48]: I thinks its fine have one value as default and changing upon another.

**Review by rajeshreeputra — APPROVED** [review]

**Review by deepakmishra2 — APPROVED** [review]

**Pull request merged**